Library/Homebrew/cmd/doctor.rb

```diff
@@ -862,10 +862,12 @@ def __check_git_version
     # https://help.github.com/articles/https-cloning-errors
     `git --version`.chomp =~ /git version ((?:\d+\.?)+)/
 
+    git_upgrade_cmd = Formula["git"].installed? ? "upgrade" : "install"
```

Comment:
Oh I only added that method no long ago. The thing is `Formula#installed_prefix` and `Formula#installed?` are very misleading. I wonder should we deprecate them.